lib/functions/main/build-tasks.sh

```diff
@@ -124,9 +127,27 @@ build_bootstrap() {
 #
 build_main() {
 	local _buildOnly=$1
+	local valid_buildOnly="u-boot kernel armbian-config armbian-zsh plymouth-theme-armbian armbian-firmware armbian-bsp"
```

Comment:
@The-going

1. shouldn't this constant be named `_kernel_buildOnly` - it is only used like this in line 147
2. Then constant `_valid_buildOnly` should be set to 'u-boot, kernel, armbian-config, armbian-zsh, plymouth-theme-armbian, armbian-firmware, armbian-bsp, chroot, bootstrap' to contain all artifacts supported and we should add a check - maybe in a separate function `build_validate_buildOnly $_valid_buildOnly $_buildOnly` to fail, if `_buildOnly` conatins anything not matching an artifact name of `$_valid_buildOnly`
3. I'd prefer to let the local variable names start with an `_` to reflect the local visibility - I know this is a convention not yet used consequently, or did I miss a coding convention you apply already for this?

Reply:
@The-going I can take care of adding this, if you want.